src/Autocompletion.ts

```diff
@@ -12,11 +12,12 @@ export default class Autocompletion implements i.AutocompletionProvider {
         let specializedProviders = PluginManager.specializedAutocompletionProviders(job.prompt.expandedFinishedLexemes);
         let providers = specializedProviders.length ? specializedProviders : PluginManager.genericAutocompletionProviders;
 
-        // FIXME: skip suggestions that have 0 score.
         return Promise.all(providers.map(provider => provider.getSuggestions(job))).then(results =>
             _._(results)
                 .flatten()
-                .filter((suggestion: Suggestion) => !suggestion.shouldIgnore(job))
+                .filter((suggestion: Suggestion) =>
+                    !suggestion.shouldIgnore(job) &&
+                    score(suggestion.value, suggestion.getPrefix(job)) > 0)
```

Comment:
I should have made the comment clearer, but the thing is, scoring is quite time consuming, and we'd like to avoid doing it twice.